tensorflow/python/kernel_tests/constant_op_test.py

```diff
@@ -768,6 +768,12 @@ def testNoShape(self):
       self.assertAllEqual(
           [[3, 3], [3, 3]], a.eval(feed_dict={p: [[3, 3], [3, 3]]}))
 
+  def testGradient(self):
+    with self.test_session():
+      x = array_ops.placeholder(dtypes_lib.float32, [5, 7])
+      y = array_ops.placeholder_with_default(x, None)
+      err = gradient_checker.compute_gradient_error(x, [5, 7], y, [5, 7])
+      self.assertLess(err, 1e-3)
```

Comment:
given that this should be passing the identity, should we be even more aggressive and expect fewer differnces?